Sources/SwiftUIKit/Views/Label.swift

```diff
@@ -22,6 +22,17 @@ public class Label: UILabel {
         accessibility(label: text, traits: .staticText)
     }
     
+    public init(_ attributedText: AttributedString) {
+        super.init(frame: .zero)
+        
+        self.attributedText = attributedText
+        
+        if #available(iOS 10.0, *) {
+            adjustsFontForContentSizeCategory = true
```

Comment:
should this be toggle-able?

Reply:
You can configure this until there is a reported issue.